lib/resources/dstu2/document_reference_create.yaml

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
I would be concerned that tracking cardinality values different than the official spec would end up causing more confusion and we'd have to fend off attempts to 'correct' the massaged cardinalities both from our future-selves and well-intentioned consumers. It won't be clear without actually looking at the code if we have intentionally changed the cardinality due to implementation, or if it is in fact just a typo. I hesitated to list the cardinalities due to this reason. 

Another tact is to consider what they actually mean to us in practice; whether the field is optional or required, and whether it is a list.  The Required column already implies the cardinality of a single valued field. We could just replace the cardinality cell with whether it is a list or not.